### ISSUE TITLE
Import layers as priority areas

### ISSUE THREAD

**Issue by severinmenard** [hotosm/osm-tasking-manager2#558]
Would be interesting to be able, not only to draw, but to import existing priority layers (eg a flood extent), as geojson or kml files

**Comment by pedrito1414** [comment]
Seconded. As MSF priorities change, it would be very useful to be able to update the task with new priorities for mappers.

**Comment by pellis2** [comment]
Agreed! This would add a lot more flexibility to the tasking manager.
